api/update-docker.sh

```diff
@@ -8,4 +8,4 @@ fi
 
 echo "Running new process"
 
-docker run --rm -t -p 80:8080 jemisonf/msgr_api:latest &
\ No newline at end of file
+docker run --rm -t -d -p 80:5000 -v /home/ec2-user/cs340_final_project/conf/configuration.py:/app/configuration.py jemisonf/msgr_api:latest
```

Comment:
From our discussion, it might be helpful to add `--name msgrdb` here so that only one instance ends up getting created and so we can easily keep track of it

Reply:
The source location of the volume is still referencing a `conf` directory. I think this needs to get removed so the source path is just `/home/ec2-user/cs340_final_project/configuration.py`

Reply:
Not a big deal, but could you also add a newline to the end of the file?